gum/gumeventsink.c

```diff
@@ -60,3 +79,40 @@ gum_event_sink_stop (GumEventSink * self)
   if (iface->stop != NULL)
     iface->stop (self);
 }
+
+static void gum_default_event_sink_class_init (GumDefaultEventSinkClass* klass)
```

Comment:
```suggestion
static void
gum_default_event_sink_class_init (GumDefaultEventSinkClass * klass)
```

(Here and elsewhere.)